src/libstd/panicking.rs

```diff
@@ -19,7 +19,7 @@ use rt::{backtrace, unwind};
 use sys::stdio::Stderr;
 use thread;
 
-// Defined in this module instead of old_io::stdio so that the unwinding
+// Defined in this module instead of `io::stdio` so that the unwinding
```

Comment:
@alexcrichton this comment looks half-missing - it was first added in 5ec36c358f74fe83332231e774ea20a21d165120. Any idea what it should say?